### DISCUSSION TITLE
Adding additional Historical records table

### DISCUSSION THREAD

**Discussion by Brooksey07** [poblabs/weewx-belchertown#922] [Ideas]
I have seen something from a local PWS that has various dynamic tables that show the various records for each day throughout the year.

![image](https://github.com/poblabs/weewx-belchertown/assets/156317406/f5a859f5-1406-4b4a-9ba1-a77e711aa30a)

Is this sort of thing achievable using high charts, not that there's anything wrong with the current records page, I just think it would be cool if we could display daily records.

**Comment by roe-dl** [comment]
I guess you don't need HighCharts for that. A WeeWX template together with some JavaScript code will do, too.
